Public/index.html

```diff
@@ -28,11 +28,15 @@
         <section class="">
             <div class="container">
                 <div class="row" id="">
-
+                  <h1>log in </h1>
+                   <form id="logInForm" class ="logInForm">
```

Comment:
what the purpose of adding id and class for all elements ?

Reply:
because I'll use them later